### ISSUE TITLE
Maximize, close controls are not accessible using keyboard present under 'Save meta data' screen: A11y_WinForm.NETRuntime_Keyboard

### ISSUE THREAD

**Issue by shindepra** [dotnet/winforms#13753]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags:
#A11ySev2; #A11yTCS; #WCAG2.1.1; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Keybaord; #A11yMAS

### Environment Details:
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061  

### Repro Steps:

1. Open SQL Server Migration Assistant for oracle.  
2. Splash screen appears and SSMA is launched 
3. Navigate to the 'File' menu and select it. 
4. Navigate to 'save project', 'close' menu item and select it. 
5. Navigate all controls on 'Save metadata' dialog. 
6. Verify whether maximize, close controls are accessible using keyboard or not.

### Actual Result:
Maximize, close controls are not accessible using keyboard present under 'Save meta data' screen. 

### Expected Result:
Maximize, close controls should be accessible using keyboard present under 'Save meta data' screen. 

### User Empact:
Keyboard users will be affected as they miss using the feature and the functionality of the controls if the keyboard focus doesn't land on the interactive elements on the page. Keyboard users cannot interact and miss accessing the controls.

Sample App used:-

[RadioButtonFocusDemo.zip](https://github.com/user-attachments/files/21483118/RadioButtonFocusDemo.zip)

### Attachment:

<img width="843" height="503" alt="Image" src="https://github.com/user-attachments/assets/8a312192-29b2-4fb8-aee8-b5c59f840679" />

https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-kkumar2_microsoft_com/Ea-djD821KlOkygZnpfWD9EBx8vKbxETN9_aqqVq0Mu7dg?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=GjqZrf

**Comment by LeafShi1** [comment]
This is not a Winform bug. The TabStop property controls whether a control can receive focus via the Tab key. Title bar buttons are not controls and are therefore not controlled by TabStop. Moreover, these buttons belong to the non-client area and are usually controlled by the Windows operating system.  

Although users can't focus these buttons using the Tab key, they can still maximize and minimize the window using shortcut keys, such as Windows+Up and Windows+Down.
